### PR TITLE
Monitor Cmdlet 1.2.2 changes 

### PR DESCRIPTION
**Pull request by nithinpnp**
## Description

* New action group receivers added for action group
	-ItsmReceiver
	-VoiceReceiver
	-ArmRoleReceiver
	-AzureFunctionReceiver
	-LogicAppReceiver
	-AutomationRunbookReceiver
	-AzureAppPushReceiver

* Use common alert schema enabled for the receivers. This is not applicable for SMS, Azure App push , ITSM and Voice recievers
* Webhooks now supports Azure active directory authentication .

## Checklist

- [] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by cormacpayne** [comment]
@gucalder Hey Guillermo, would you mind taking a look at this PR?

**Comment by cormacpayne** [comment]
@nithinpnp ping on the above comments

**Comment by cormacpayne** [comment]
Closing due to lack of activity. Please feel free to re-open this PR whenever you have a chance to address the above comments.

File: src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs [hunk 1]
```diff
@@ -40,6 +54,14 @@ public class NewAzureRmActionGroupReceiverCommand : AzureRMCmdlet
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
+        /// <summary>
+        /// Gets or sets the UseCommonAlertSchema parameter
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The flag whether to use common alert schema . This value will be neglected" +
+            "for SMS, Azure App push , ITSM and Voice recievers.")]
+        [ValidateNotNullOrEmpty]
+        public bool UseCommonAlertSchema { get; set; } = false;
```
[the remark this answers is not in the document]
Comment (ritwik8119) [reply src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs]: Regenerated the help file.

[the remark this answers is not in the document]
Comment (ritwik8119) [reply src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs]: Updated.

File: src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs [hunk 2]
```diff
@@ -80,12 +102,208 @@ public class NewAzureRmActionGroupReceiverCommand : AzureRMCmdlet
...
         [Parameter(ParameterSetName = NewWebhookReceiver, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The address of the webhook receiver")]
         [ValidateNotNullOrEmpty]
         public string ServiceUri { get; set; }
 
+        /// <summary>
+        /// Gets or sets the UseAadAuth parameter
+        /// </summary>
+        [Parameter(ParameterSetName = NewWebhookReceiver, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "the flag to use add auth")]
+        [ValidateNotNullOrEmpty]
+        public bool UseAadAuth { get; set; } = false;
```
[the remark this answers is not in the document]
Comment (ritwik8119) [reply src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs]: Updated.

File: src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs [hunk 3]
```diff
@@ -80,12 +102,208 @@ public class NewAzureRmActionGroupReceiverCommand : AzureRMCmdlet
...
+        [Parameter(ParameterSetName = NewAutomationRunbookReceiver, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "the RunbookName")]
+        [ValidateNotNullOrEmpty]
+        public string RunbookName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the IsGlobalRunbook parameter
+        /// </summary>
+        [Parameter(ParameterSetName = NewAutomationRunbookReceiver, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = " indicating whether this instance is global runbook")]
+        [ValidateNotNullOrEmpty]
+        public bool IsGlobalRunbook { get; set; } = false;
```
[the remark this answers is not in the document]
Comment (ritwik8119) [reply src/Monitor/Monitor/ActionGroups/NewAzureRmActionGroupReceiverCommand.cs]: Updated.

**Comment by nithinpnp** [comment]
@cormacpayne could you please review .

**Pull request merged**